Source/i_sound.c

```diff
@@ -34,8 +34,10 @@
 
```

Comment:
The changes to `i_sound.c` are massive. Please note that I took great care to only carefully modify this file and keep its original code structure intact as far as possible. I am doing this out of respect for the original authors of MBF and WinMBF, respectively. 

Whatever, it seems to me that you brute-forced the entire implementation from Crispy Doom into this file. I'd prefer if you could find out which changes are actually needed to get midiproc running and apply only those.